Automattic/jetpack
# RNA: Add ActionPopover component

Part of #27292

#### Changes proposed in this Pull Request:

* Adds the `ActionPopover` component

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Open the Storybook with `( cd projects/js-packages/storybook && pnpm run storybook:dev )`
* Go to `JS Packages/Components/Action Popover`
* Check that it complies with the component needed on the linked issue

TTzDk3dzct551hYWRXPFa3-fi-3062%3A30275&t=l1snOM47GOlP7xA1-4

![2022-11-29_11-57-08](https://user-images.githubusercontent.com/8486249/204563171-705e96ab-5081-4b42-8e02-30ed81714303.png)
